src/model.ts

```diff
@@ -13,7 +13,7 @@ export { InternalStorage, UserType };
  * @param pars Parameters to pass to the model
  *
  */
```

Comment:
@param comment for the `unusedUserAction` param?

Reply:
thanks, added now